src/pages/index.js

```diff
@@ -17,6 +17,9 @@ import { RequisitionsPage } from './RequisitionsPage';
 import { RequisitionPage,
          checkForFinaliseError as checkForRequisitionFinaliseError,
        } from './RequisitionPage';
+import { SupplyRequisitionPage,
+         checkForFinaliseError as checkForSupplierRequisitionError,
```

Comment:
Can `checkForError:` be undefined in FINALISABLE_PAGES ? I updated assuming so, will check tomorrow